server/index.js

```diff
@@ -17,9 +17,13 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-
 'use strict';
 
+var fs = require('fs');
+var path = require('path');
+
+var TChannelAsThrift = require('tchannel/as/thrift');
```

Comment:
It is now possible to extract the .AsThrift constructor from a channel instance.